src/components/Footer/Footer.styles.js

```diff
@@ -15,6 +16,14 @@ export const FooterLinkContainer = styled.span`
   font-size: 14px;
 `;
 
+export const Anchor = styled.a`
```

Comment:
You can remove this when you use the styled-components `as` prop instead like this

```
<Link as="a" href="https://ooloo.io" target="_blank" rel="noopener noreferrer" />
```